src/TeacherDashboard/TeacherDashboard.js

```diff
@@ -1,11 +1,26 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import './TeacherDashboard.scss'
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { getLessons } from "../thunks/getLessons";
+import { getStudents } from "../thunks/getStudents"
+import { getTeacher } from "../thunks/getTeacher"
 
-const TeacherDashboard = () => {
+const TeacherDashboard = (props) => {
+
+  useEffect (() => {
+    async function fetchData() {
+      await props.getTeacher()
+      await props.getStudents()
+      await props.getLessons()
+    } 
+    fetchData()
+    // eslint-disable-next-line 
+  }, [])
```

Comment:
I'm not thrilled about the //eslint-disable right here - but I wanted to get this branch pushed up so ya'll could start on your pages.  The warning asked us to desctructor props, but I was having a few issues, I'll come back to this before final submission.

Reply:
I had to do this too, I'll try to read up more on this as well